.bcr/presubmit.yml

```diff
@@ -0,0 +1,10 @@
+bcr_test_module:
+  module_path: "examples/naming_package_files"
```

Comment:
`bcr_test_module` gives a stronger guarantee: the module works when depended on by an external repository.

The current testing for rules_pkg just tests within the rules_pkg workspace, which won't catch as many bugs, for example if the `standard_package` filegroup is missing an entry, the current tests would pass, but this `bcr_test_module` would fail.

See https://github.com/bazelbuild/continuous-integration/issues/1302 by @meteorcloudy for more info

Reply:
I'll recommend to have both. Since the `bcr_test_module` doesn't actually download and patch the module through Bazel, it prepares the working directory via [some custom python logic](https://github.com/bazelbuild/continuous-integration/blob/91f3d02acd9ff54f8f9e34e5a7426b4254456925/buildkite/bazel-central-registry/bcr_presubmit.py#L226-L242), which might miss some issue caused by weird patches.

Reply:
I'll add that in a separate PR, let's get the automation going matching the existing files.